example/src/Demo.jsx

```diff
@@ -14,58 +14,101 @@ require('../../example/swagger-files/response-schemas.json');
 
 require('../../packages/api-logs/main.css');
 
-function Demo({ fetchSwagger, status, docs, oas, oauth }) {
-  return (
-    <div>
-      <div className="api-list-header">
-        <ApiList fetchSwagger={fetchSwagger} />
-        <pre>{status.join('\n')}</pre>
+class Demo extends React.Component {
```

Comment:
You could have used the `useState()` hook for this if you wanted to keep it as a functional component. https://reactjs.org/docs/hooks-state.html I'm completely fine with this approach though.

Reply:
Ended up reshuffling this addition around a bit so we can easily add new "experiment" toggles. Going to keep it as-is since I don't think setting up hooks dynamically will work with this kind of set up. Happy to revisit later as a foray into hooks though!